docs/products/flink/howto/real_time_alerting_solution.rst

```diff
@@ -116,13 +116,13 @@ This setup uses a fixed threshold to filter any instances of high CPU load to a
 
 #. Go to the **Data Tables** subtab.
 
-#. Select your Kafka service, enter ``CPU_IN`` as the name, select ``cpu_load_stats_real`` as the topic, and enter the following as the SQL schema, then click **Create Table**:
+#. Select your Kafka service, use the default selection for connector type, key, and value data format, enter ``CPU_IN`` as the name, select ``cpu_load_stats_real`` as the topic, and enter the following as the SQL schema, then click **Create Table**:
```

Comment:
"use the default selection for connector type" <- I'm not sure which is the default selection, could be mention the "standard Kafka connector type" and JSON for key and value data format?